package.json

```diff
@@ -52,19 +57,18 @@
   "devDependencies": {
     "@babel/core": "^7.14.6",
     "@ladle/react": "^2.4.5",
-    "@storybook/addon-a11y": "7.5.3",
+    "@storybook/addon-a11y": "7.6.0-alpha.7",
```

Comment:
I'll change these to the stable version once it has been released, while also removing the resolutions.